asyncio_redis_rate_limit/__init__.py

```diff
@@ -95,7 +99,17 @@ async def _acquire(self) -> None:
         pipeline = self._backend.pipeline()
 
         async with self._lock:
-            current_rate = await self._run_pipeline(cache_key, pipeline)
+            redis_version = await self.redis_version()
```

Comment:
Can we do this earlier? We don't need to query this every time. We only need this once per app.